### PR TITLE
Marko/Gaia Documentation Fixes

### PR DESCRIPTION
**Pull request by tac0turtle**
Fixed documentation discrepencies with Gaia and Cosmos-SDK

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Older releases up to 0.34.x are available from the [Cosmos SDK releases page](ht
 mkdir -p $GOPATH/src/github.com/cosmos
 cd $GOPATH/src/github.com/cosmos
 git clone -b <latest-release-tag> https://github.com/cosmos/gaia
-cd cosmos-sdk && make tools install
+cd gaia && make tools install
```
Comment (alessio) [inline docs/installation.md]: IIRC `tools` is not actually required anymore to install the binaries

**Review by alessio — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
+## What is Gaia?
+
+- [Gaia](./what-is-gaia.md)
```
Comment (alexanderbez) [inline docs/README.md:7]: Where is this located? I don't see it in docs?

EDIT: is this in the base branch?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+     2. ensure that nobody else has already begun working on this issue, if they have
+       make sure to contact them to collaborate,
+     3. if nobody has been assigned the issue and you would like to work on it
+       make a comment on the issue to inform the community of your intentions
+       to begin work,
+     4. follow standard github best practices: fork the repo, branch from the
+       top of `master`, make some commits, and submit a PR to `master`,
+         - for core developers working within the Gaia repo, 
+	 to ensure a clear ownership of branches, branches must be named with the convention `yourname/{issue-}feature-name`.  
+     5. include `WIP:` in the PR-title to and submit your PR early, even if it's
```
Comment (alexanderbez) [inline CONTRIBUTING.md]:
```suggestion
     5. submit your PR early and make sure it's opened as a `Draft`, even if it's
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+       incomplete, this indicates to the community you're working on something and
+       allows them to provide comments early in the development process. When the code
+       is complete it can be marked as ready-for-review by replacing `WIP:` with
```
Comment (alexanderbez) [inline CONTRIBUTING.md]:
```suggestion
       is complete it should be marked as ready-for-review using Github's `Mark Ready` feature.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+       `R4R:` in the PR-title.
```
Comment (alexanderbez) [inline CONTRIBUTING.md]:
```suggestion
```

**Review by alexanderbez — CHANGES_REQUESTED** [review]
Thanks @marbar3778! Left a few tidbits.

File: docs/DOCS_README.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+much as possible with its [counterpart in the Tendermint Core repo](https://github.com/tendermint/tendermint/blob/develop/docs/DOCS_README.md).
+
+### Update and Build the RPC docs
+
+1. Execute the following command at the root directory to install the swagger-ui generate tool.
+    ```bash
+    make tools
+    ```
+2. Edit API docs
+    1. Directly Edit API docs manually: `client/lcd/swagger-ui/swagger.yaml`.
```
Comment (alexanderbez) [inline docs/DOCS_README.md]:
```suggestion
    1. Directly Edit API docs manually: `cmd/gaiacli/swagger-ui/swagger.yaml`.
```

File: docs/DOCS_README.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    2. Edit API docs within the [Swagger Editor](https://editor.swagger.io/). Please refer to this [document](https://swagger.io/docs/specification/2-0/basic-structure/) for the correct structure in `.yaml`.
+3. Download `swagger.yaml` and replace the old `swagger.yaml` under fold `client/lcd/swagger-ui`.
```
Comment (alexanderbez) [inline docs/DOCS_README.md]:
```suggestion
3. Download `swagger.yaml` and replace the old `swagger.yaml` under fold `cmd/gaiacli/swagger-ui`.
```

File: docs/README.md (hunk 2 above)
Comment (tac0turtle) [reply docs/README.md:7]: yes, this would be in alessio's branch alessio/docs

**Review by alexanderbez — APPROVED** [review]

**Pull request merged**